<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KernelProcessProtectionModderClass</class>
 <widget class="QWidget" name="KernelProcessProtectionModderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>297</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Protection Modder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pidLabel">
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pidLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="unprotectButton">
       <property name="text">
        <string>Unprotect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="protectButton">
       <property name="text">
        <string>Protect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KernelProcessProtectionModder.qrc"/>
 </resources>
 <connections/>
</ui>
